clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -213,9 +212,7 @@ public DaVinciBackend(
           true,
           functionToCheckWhetherStorageEngineShouldBeKeptOrNot(managedClients));
       storageService.start();
-      PubSubClientsFactory pubSubClientsFactory = configLoader.getVeniceServerConfig().getPubSubClientsFactory();
-      VeniceWriterFactory writerFactory =
-          new VeniceWriterFactory(backendProps.toProperties(), pubSubClientsFactory.getProducerAdapterFactory(), null);
+      VeniceWriterFactory writerFactory = new VeniceWriterFactory(backendProps.toProperties(), null);
```

Comment:
This change creates a new `PubSubClientsFactory` object which is not desirable IMHO.  Besides, in https://github.com/linkedin/venice/pull/1705 we're removing VW instantiation here altogether.